docker/etc/nginx/sites-enabled/02_mapbox_proxy_cache.conf

```diff
@@ -0,0 +1,6 @@
+proxy_cache_path /var/cache/nginx/mapbox
```

Comment:
This will be cleared on each deploy, which *may* not be what we want.

Reply:
Thanks. Yes, that was my assumption and I thought that having the cache cleared on deploy was a good idea. Some kind of easy purge mechanism...